packages/cli/src/Server.ts

```diff
@@ -403,19 +367,60 @@ export class Server extends AbstractServer {
 			this.app.use('/icons/@:scope/:packageName/*/*.(svg|png)', serveIcons);
 			this.app.use('/icons/:packageName/*/*.(svg|png)', serveIcons);
 
+			const isTLSEnabled = this.protocol === 'https' && !!(this.sslKey && this.sslCert);
+			const securityHeadersMiddleware = helmet({
+				contentSecurityPolicy: false,
+				xFrameOptions: { action: 'sameorigin' },
```

Comment:
I believe this breaks the workflow preview on n8n.io. The `n8n-demo` component is rendered in an `iframe` that is served from another domain. Should we add some way to override this for the preview service? The CORS header might also affect this

Reply:
ah yes. I'll add a check for that. But, I really hope that we stop using a full n8n instance for the preview in the near future.